### ISSUE TITLE
Accessibility - Dataverse - Document Landmark

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#90]
### Homepage: https://data.qdr.syr.edu/
- [x] Whole Page: 
![image](https://user-images.githubusercontent.com/89411909/134693541-62726dd2-f080-4603-b9bb-378596398e5f.png)

### Advanced Search: https://data.qdr.syr.edu/dataverse/main/search
- [x] Same as Above

**Comment by qqmyers** [comment]
Should be fixed.
